pkg/kube/resourcemapper.go

```diff
@@ -0,0 +1,11 @@
+package kube
+
+import (
+	"k8s.io/api/extensions/v1beta1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+// ResourceMap are resources from where changes are going to be detected
+var ResourceMap = map[string]runtime.Object{
+	"deployments": &v1beta1.Deployment{},
```

Comment:
You're watching only deployments. Watch other resources too

Reply:
getting some api related issues while handling daemonsets and statefulsets. Will resolve and handle in next version.